project/Merging.scala

```diff
@@ -7,7 +7,7 @@ object Merging {
     case PathList("google", "protobuf", _ @ _*) => MergeStrategy.first
     case PathList("META-INF", "versions", "9", "module-info.class") => MergeStrategy.first
     case "module-info.class" =>
-      MergeStrategy.discard // JDK 8 does not use the file module-info.class so it is safe to discard the file.
+      MergeStrategy.discard // JDK 11 does not use the file module-info.class so it is safe to discard the file.
```

Comment:
I believe module-info was introduced in Java 9, I don't believe this comment is true any more

Reply:
Ah! Thank you! I couldn't find info on it!